actions/install/README.md

```diff
@@ -0,0 +1,29 @@
+# slsa-verifier-installer GitHub Action
```

Comment:
Add a section under the main README that link to this README for installing the action

Reply:
Done